server/reload_test.go

```diff
@@ -2603,6 +2603,23 @@ func TestConfigReloadAccountUsers(t *testing.T) {
 		t.Fatalf("Error on subscribe: %v", err)
 	}
 
+	// confirm subscriptions before and after reload.
+	var expectedSubs uint32 = 4
+	sAcc, _ := s.LookupAccount("synadia")
+	sAcc.mu.RLock()
```

Comment:
Capture err and do require_NoError(t, err)

Reply:
thanks, updated